transcription/transcription.go

```diff
@@ -0,0 +1,31 @@
+package transcription
+
+import (
+	"net/smtp"
+	"strings"
+)
+
+// SendEmail authenticates, creates message, and sends email from arguments.
+func SendEmail(username string, password string, host string, port int, to []string, subject string, body string) error {
```

Comment:
Can we document what some of these arguments mean? I don't think it's self-explanatory what `host` or `port` mean.